code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -305,4 +382,99 @@
 /obj/effect/hotspot/singularity_pull()
 	return
 
+/datum/looping_sound/fire
+	mid_sounds = list('sound/effects/fireclip1.ogg' = 1, 'sound/effects/fireclip2.ogg' = 1, 'sound/effects/fireclip3.ogg' = 1, 'sound/effects/fireclip4.ogg' = 1,
+	'sound/effects/fireclip5.ogg' = 1, 'sound/effects/fireclip6.ogg' = 1, 'sound/effects/fireclip7.ogg' = 1)
+	volume = 100
+	mid_length = 2 SECONDS
+	falloff_distance = 1
+//handle the grouping of hotspot and then determining an average center to play sound in
+/datum/hot_group
```

Comment:
Use three slashes for the autodoc, not two. Also properly document the vars for this datum please.